cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -85,6 +86,8 @@ impl CincinnatiGraphFetchPlugin {
         };
 
         let client = reqwest::ClientBuilder::new()
+            .gzip(true)
+            .timeout(Duration::from_secs(30))
```

Comment:
This should be configurable on the plugin level

Reply:
Good idea

Reply:
Implemented this in f0411e877f65b9c3eec473d714c8ec5d2c3cd9df although I'm not sure if that is sufficient

Reply:
That should so!